readers/api/endpoint_test.go

```diff
@@ -784,8 +827,9 @@ func TestReadAll(t *testing.T) {
 			key:    userToken,
 			status: http.StatusOK,
 			res: pageRes{
-				Total:    uint64(len(messages)),
-				Messages: messages[0:10],
+				PageMetadata: readers.PageMetadata{Limit: 10, Format: "messages", Interval: ""},
```

Comment:
Should this be `10h`?
```suggestion
				PageMetadata: readers.PageMetadata{Limit: 10, Format: "messages", Interval: "10h"},
```

Reply:
Yes